tests/command.conflicts.test.js

```diff
@@ -0,0 +1,92 @@
+const commander = require('../');
+
+describe('command with conflicting options', () => {
+  function makeProgram() {
+    const actionMock = jest.fn();
+    const program = new commander.Command();
+    program
+      .exitOverride()
```

Comment:
To avoid the error messages polluting the test display, can suppress the output using:
```js
      .configureOutput({
        writeErr: () => {}
      })
```

Reply:
Great tip! Done in https://github.com/tj/commander.js/pull/1678/commits/351ef38144c0eae579ba2365b011117d3082067e